capa/ida/helpers.py

```diff
@@ -34,6 +37,10 @@
 # arch type as returned by idainfo.procname
 SUPPORTED_ARCH_TYPES = ("metapc",)
 
+CAPA_NETNODE = "$ com.mandiant.capa"
```

Comment:
```suggestion
CAPA_NETNODE = "$ com.mandiant.capa.v1"
```

see below discussion on versioning

Reply:
and/or use `capa.version.__version__` directly in the path, and then we can rely on each release of capa to be namespaced.